misc/atrium-specs/src/commonMain/kotlin/ch/tutteli/atrium/specs/integration/MapToContainInOrderOnlyKeyValueExpectationsSpec.kt

```diff
@@ -257,6 +256,87 @@ abstract class MapToContainInOrderOnlyKeyValueExpectationsSpec(
 
         context("report options") {
             //TODO #1129 add tests, see IterableToContainInOrderOnlyValuesExpectationsSpec -> report options
+            context("") {
+                it("shows only failing with report option `showOnlyFailing`") {
+                    expect {
+                        expect(map).toContainFun(
+                            keyValue("a") { toEqual(1) },
+                            keyValue("b") { toEqual(2) },
+                            keyValue("c") { toEqual(3) },
+                            report = { showOnlyFailing() }
+                        )
+                    }.toThrow<AssertionError> {
+                        message {
+                            toContainSize(2, 3)
+                            elementSuccess(0, "1", "a", "1")
+                            elementFailing(0, "2", "a", "1")
+                            elementOutOfBound(3, "d", "4")
+                        }
+                    }
+                }
+                it("shows only failing with report option `showOnlyFailingIfMoreExpectedElementsThan(2)` because there are 3") {
+                    expect {
+                        expect(map).toContainFun(
+                            keyValue("a") { toEqual(1) },
+                            keyValue("b") { toEqual(2) },
+                            keyValue("c") { toEqual(3) },
+                            report = { showOnlyFailingIfMoreExpectedElementsThan(2) }
+                        )
+                    }.toThrow<AssertionError> {
+                        message {
+                            elementSuccess(0, "1", "a", "1")
+                            elementFailing(0, "2", "a", "1")
+                            elementOutOfBound(3, "d", "4")
+                            toContainSize(2, 3)
+                        }
+                    }
+                }
+                it("shows summary with report option `showOnlyFailingIfMoreExpectedElementsThan(2)` because there are 2") {
+                    expect {
+                        expect(map).toContainFun(
+                            keyValue("a") { toEqual(1) },
+                            report = { showOnlyFailingIfMoreExpectedElementsThan(1) }
```

Comment:
Didn't see this before, you should do what you describe in `it` i.e. pass 2 and not 1 to showOnlyFailingIfMoreExpectedElementsThan
```suggestion
                            report = { showOnlyFailingIfMoreExpectedElementsThan(2) }
```
Accordingly, you need to add two entry expectations to `toContainFun` to meet the criteria

---

Alternatively you can rewrite the `it` to:
```
it("shows summary with report option `showOnlyFailingIfMoreExpectedElementsThan(1)` because there are 1") {
```